src/RelationTag.h

```diff
@@ -58,7 +58,6 @@ enum class RelationRepresentation {
     BTREE,         // use btree data-structure
     BTREE_DELETE,  // use btree_delete data-structure
     EQREL,         // use union data-structure
-    PROVENANCE,    // use custom btree data-structure with provenance extras
```

Comment:
Provenance is not needed anymore?

Reply:
The fact that a relation must have auxiliary arities to store provenance information does not mean it has to be a RelationRepresentation. In practice it's just a Btree. 
Now that we have lattice arguments, these lattice arguments are also stored as auxiliary arities.